tests/Behat/Context/Setup/AdminUserContext.php

```diff
@@ -30,4 +47,50 @@ public function thisAdministratorHasRole(
 
         $this->administratorManager->flush();
     }
+
+    /**
+     * @When I want to add a new administrator :email named :firstName last name :lastname and locale :locale
+     */
+    public function iWantToAddAnNewAdministratorNamedLastName(string $email, string $firstName, string $lastname, string $locale): void
+    {
+        $adminUser = $this->adminUserExampleFactory->create([
+            'email' => $email,
+            'username' => $email,
+            'password' => $email,
+            'enabled' => true,
+            'locale_code' => $locale,
+            'first_name' => $firstName,
+            'last_name' => $lastname,
+        ]);
+
+        $this->adminUserRepository->add($adminUser);
+        $this->sharedStorage->set('administrator', $adminUser);
+    }
+
+    /**
+     * @Then I should seeing administrator account :email named :firstName last name :lastname and locale :locale
+     */
+    public function iShouldSeeAdministratorAccountNamedLastNameAndLocale(string $email, string $firstName, string $lastname, string $locale): void
```

Comment:
This should not be placed in the `Setup` context - as well as the check should be done with UI